File: ObjectRequestBrokerJava/src/main/java/examples/math/MathServer.java
package examples.math;

import orbapi.ORB;

public class MathServer implements IMath {
    private static final int MATH_SERVER_PORT = 1010;
    private static final String MATH_SERVER_NAME = "MathServer";

    @Override
    public double do_add(double a, double b) {
        return a+b;
    }

    @Override
    public double do_sqr(double a) {
        return Math.sqrt(a);
    }

    public static void main(String[] args) {
        MathServer mathServer = new MathServer();
        ORB.register(MATH_SERVER_NAME,MATH_SERVER_PORT,"math",mathServer);

    }
}

File: ObjectRequestBrokerCS/ServerExecutable/MathServer.cs
using System;
using ORB.orbapi;

namespace ServerExecutable
{
    public class MathServer:IMath
    {
        private static readonly int MATH_SERVER_PORT = 1010;
        private static readonly string MATH_SERVER_NAME = "MathServer";

        
        public double do_add(double a, double b) {
            return a+b;
        }

        
        public double do_sqr(double a) {
            return Math.Sqrt(a);
        }
        
        
        public static void start(){
            var m = new MathServer();
            ORBMiddleware.Register(MATH_SERVER_NAME,MATH_SERVER_PORT,"math",m);
        }
    }
    

}